### PR TITLE
Sw 5291 edit pii details

### PR DESCRIPTION
**Pull request by Nicholassully**

### CODE REVIEW

**Review by ramchale — DISMISSED** [review]

File: internal/server/firm_deputy_hub.go [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
-	XSRFToken   string
-	Error       string
-	Errors      sirius.ValidationErrors
-	FirmDetails sirius.FirmDetails
+	Path           string
+	XSRFToken      string
+	Error          string
+	Errors         sirius.ValidationErrors
+	FirmDetails    sirius.FirmDetails
+	Success        bool
```
Comment (denialanderror) [inline internal/server/firm_deputy_hub.go:21]: Not sure I see the point of having a `Success` boolean and a `SuccessMessage`. The presence of a message is the indication of success.

File: web/template/manage-pii-details.gotmpl [hunk 2]
```diff
@@ -0,0 +1,172 @@
+{{ template "page" . }}
```
Comment (denialanderror) [inline web/template/manage-pii-details.gotmpl:1]: This template needs redoing. there are a lot of injected tags, e.g. data-dashlane-id that shouldn't be there.

**Review by denialanderror — CHANGES_REQUESTED** [review]

File: internal/server/manage_pii_details.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+			return Redirect(fmt.Sprintf("/%d?success=piiDetails", firmId))
+
+		default:
+			return StatusError(http.StatusMethodNotAllowed)
+		}
+	}
+}
+
+func renameEditPiiValidationErrorMessages(siriusError sirius.ValidationErrors) sirius.ValidationErrors {
```
Comment (denialanderror) [inline internal/server/manage_pii_details.go:82]: You can do all this with a translate function in the template. Take a look at the work I did on Pro Hub.

File: json-server/routes.json [hunk 4]
```diff
@@ -1,4 +1,5 @@
 {
     "/api/v1/*": "/$1",
-    "/firms/:id": "/firms"
+    "/firms/:id": "/firms",
+    "/firms/:firmId/editPii": "/edit"
```
Comment (denialanderror) [inline json-server/routes.json:4]: There isn't a route with "editPi" called in the code and there isn't an "edit" object in the db either

**Review by denialanderror — CHANGES_REQUESTED** [review]

**Review by denialanderror — APPROVED** [review]

**Pull request merged**